Source/Build Specs/ni_protobuf_types.vipb

```diff
@@ -0,0 +1,187 @@
+<VI_Package_Builder_Settings Version="2020.1" Created_Date="2023-01-23 14:05:21" Modified_Date="2023-11-16 10:58:35" Creator="lvadmin" Comments="" ID="7205f398b8afd492ca4d6a2f13d92a99">
+  <Library_General_Settings>
+    <Package_File_Name>ni_protobuf_types</Package_File_Name>
+    <Library_Version>1.0.0.8</Library_Version>
+    <Auto_Increment_Version>false</Auto_Increment_Version>
+    <Library_Source_Folder>..\protobuf\ni.protobuf.types</Library_Source_Folder>
```

Comment:
@dixonjoel This directory doesn't exist:

![image](https://github.com/ni/measurementlink-labview/assets/17914399/0ec20b81-9872-4491-a80c-f3e00b16ca48)

Did you move this `.vipb` file or have a stale `Source\ni.protobuf.types` directory on your hard disk?

Reply:
Yes, I moved it to https://github.com/ni/measurementlink-labview/tree/main/Source/gRPC/Generated%20APIs/ni/protobuf/types and forgot to update this. I see you have a PR to fix that.